### PR TITLE
Add Ubuntu to the main list of operating systems

### PR DESCRIPTION
**Pull request by stefwalter**

### CODE REVIEW

**Comment by stefwalter** [comment]
@andreasn I need  help with the layout here. How can I make this look better at big sizes. All the icons seem much smaller.

**Comment by andreasn** [comment]
Will take a look

**Comment by andreasn** [comment]
Maybe something like this instead?
I gave the top 3 items the class col-sm-4 and the two bottom ones col-sm-6

screenshot from 2016-11-15 14-28-04

File: running.html [hunk 1]
```diff
@@ -128,6 +128,18 @@
               </ol>
             </div>
           </div>
+	 </div>
+
+        <div class="row">
+          <div class="col-md-12"><div class="browser-header"><p>Stable, tested, delivered in a repository:</p></div></div>
```
Comment (andreasn) [inline running.html]: Can probably remove the colon here

File: running.html [hunk 2]
```diff
@@ -170,10 +182,8 @@
           </div>
         </div>
 
-        <hr>
-
         <div class="row">
-          <div class="col-md-12"><p>Less tested and/or irregularly updated packages are available on</p></div>
+          <div class="col-md-12"><div class="browser-header"><p>Less tested and irregularly updated:</p></div></div>
```
Comment (andreasn) [inline running.html]: And here too

**Pull request closed without merging**